content/en/blog/preact-x.md

```diff
@@ -0,0 +1,150 @@
+
+### 10.20.2
+
+We have faced a number of issues where an event could bubble up to a newly inserted VNode which would result in undesired behaviour, this was fixed
+[by adding an event-clock](https://github.com/preactjs/preact/pull/4322). In the following scenario, you would click the button which sets state, the browser
+interleaves event bubbling with micro-ticks, which is also what Preact uses to schedule updates. This combination means that Preact will update the UI, meaning
+that the `<div>` will get that `onClick` handler which we'll bubble up to and invoke the `click` again toggling this state immediately off again.
+
+```jsx
+const App = () => {
+  const [toggled, setToggled] = useState(false);
+
+  return toggled ? (
+    <div onClick={() => setToggled(false)}>
+      <span>clear</span>
+    </div> 
+  ) : (
+    <div>
+      <button
+        onClick={() => setToggled(true)}
+      >toggle on</button>
+    </div>
+  )
+}
+```
+
+## Stability
+
+The above are some cherry-picked releases of things that our community received _without_ breaking changes, but there is so much more... Adding a new major
+version always leaves a part of the community behind and we don't want to do that. If we look at the Preact 8 release line we can see that there's still 100.000
+downloads in the past week, the last 8.x release was 5 years ago, just to show that a part of the community gets left behind.
+
+Stability is great, we as the Preact team love stability. We actually released multiple major features on other ecosystem projects:
+
+- [Signals](https://github.com/preactjs/signals)
+- [Async rendering](https://github.com/preactjs/preact-render-to-string/pull/333)
+- [Streaming rendering](https://github.com/preactjs/preact-render-to-string/pull/354)
+- [Prefresh](https://github.com/preactjs/prefresh)
+- [The vite preset with pre-rendering](https://github.com/preactjs/preset-vite#prerendering-configuration)
+- [A new async router](https://github.com/preactjs/preact-iso)
+- [Create Preact](https://github.com/preactjs/create-preact)
+
+We value our ecosystem and we value the extensions being built through our [`options API`](https://marvinh.dev/blog/preact-options/),
+this is one of the main drivers behind not wanting to introduce these breaking changes but instead, allow you all to benefit
+from our research without a painful migration path.
+
+This doesn't mean that Preact 11 won't happen but it might not be the thing that we initially thought it would be, instead we might just drop IE11 support and give you
+those performance improvments, all while giving you the stability of Preact X. We could instead work on a meta-framework for Node like [Fresh](https://fresh.deno.dev/) as a great
+way to write Preact, we could bet more on signals... There are a lot of options while avoiding leaving a part of the community behind.
```

Comment:
Minor rewording.

```suggestion
This doesn't mean that Preact 11 won't happen but it might not be the thing that we initially thought it would be. Instead, we might just drop IE11 support and give you
those performance improvements, all while giving you the stability of Preact X. There are many more ideas floating around and we're very interested in the wider Preact experience in the context of meta-frameworks that provide things like routing out of the box. We're exploring this angle in our vite preset as well as [Fresh](https://fresh.deno.dev/) to get a good feel what a Preact first meta framework should look like.
```